src/templates/reports/monthly/civic-liker/v2.tsx

```diff
@@ -45,6 +54,30 @@ const MonthlyReportCivicLikerV2SubscribersDescription = ({
   );
 };
 
+const MonthlyReportCivicLikerV2CreatorsFundDescription = ({
+  creatorfundingCreators,
+}: {
+  creatorfundingCreators: Array<any>;
```

Comment:
simply given the number is enough? Seem you don't use the array but only the length